zebra-utils/src/bin/zebra-checkpoints/args.rs

```diff
@@ -14,6 +14,10 @@ pub struct Args {
     #[structopt(default_value = "zcash-cli", short, long)]
     pub cli: String,
 
+    /// Start looking for checkpoints after this height
+    #[structopt(default_value = "0", short, long)]
+    pub last_checkpoint: u32,
```

Comment:
This is really good code. I hadn't realised that it could be done in so few lines.

Suggestion: Let's make the genesis special case part of the type (1/3)

Step 1: Use an optional type.
```suggestion
    /// Start looking for checkpoints after this height.
    /// If there is no last checkpoint, we start looking at the Genesis block (height 0).
    #[structopt(default_value = "None", short, long)]
    pub last_checkpoint: Option<u32>,
```